app/models/spree/ability.rb

```diff
@@ -243,7 +243,7 @@ def add_product_management_abilities(user)
       can [:admin, :index], ::Admin::DfcProductImportsController
 
       # Reports page
-      can [:admin, :index, :show], ::Admin::ReportsController
+      can [:admin, :index, :show, :create], ::Admin::ReportsController
```

Comment:
you should also update the related spec

Reply:
👍 Good point, thanks!

Reply:
Oh no! I broke that when I separated the `show` action into `show` and `create`! For me that's a reason to not magically couple controller action names with permissions via the ResourceController.